tests/Unit/Subscriber/Core/RegistratorSubscriberTest.php

```diff
@@ -104,6 +140,17 @@ public function testNodeFromRegistry()
         $this->subscriber->handleNodeFromRegistry($this->persistEvent->reveal());
     }
 
+    public function testReorderNodeFomRegistry()
```

Comment:
Currently there is no guideline to add comments for tests. IMHO If an additional comment adds value, it definitely makes sense. If it's the same like the method name, not really.